src/utils/notifyHandler.ts

```diff
@@ -0,0 +1,58 @@
+import { NOTIFY_OPTIONS } from "../constants"
+import Notify from "bnc-notify"
+import { getEtherscanLink } from "../utils/getEtherscanLink"
+
+export const notify = Notify(NOTIFY_OPTIONS)
+
+export function notifyHandler(
+  hash: string,
+  type: "Deposit" | "Withdraw" | "Swap" | "Token approval",
+): void {
+  const { emitter } = notify.hash(hash)
+
+  emitter.on("txPool", (transaction) => {
+    if (transaction.hash) {
+      return {
+        message: `${type} transaction is pending, view it <a href="${getEtherscanLink(
+          transaction.hash,
+          "tx",
+        )}" rel="noopener noreferrer" target="_blank">on Etherscan</a>.`,
```

Comment:
not sure if having an href in the modal makes sense, doesn't it open the etherscan link when you click anywhere on the modal?

Reply:
It could be that way that click the toast and open a new window. 

However, the behavior are different based on toast types: 1. Pending toast has a close button and auto-disappear after txn confirmed. 2. Failed toast has a close button and don't auto-disappear. 3. Sent txn that hasn't get into the mempool doesn't have Etherscan link. 4. Confirmed txn auto-disappear in 2s (not enough time to click anything). 

So I changed the click toast -> open Etherscan behavior to be more literal because only the pending and failed txn make sense to have Etherscan link.